packages/babel-core/test/config-chain.js

```diff
@@ -14,6 +15,24 @@ function loadOptions(opts) {
   });
 }
 
+function pairs(items) {
+  const pairs = [];
+  for (let i = 0; i < items.length - 1; i++) {
+    for (let j = i + 1; j < items.length; j++) {
+      pairs.push([items[i], items[j]]);
+    }
+  }
+  return pairs;
+}
+
+async function getTemp(name) {
+  const cwd = await fs.promises.mkdtemp(os.tmpdir() + path.sep + name);
+  const tmp = name => path.join(cwd, name);
+  const config = name =>
+    fs.promises.copyFile(fixture("config-files-templates", name), tmp(name));
```

Comment:
`fs.promises` is not supported until 10.x

Reply:
Oh right 🤦‍♂️